ext/bin/pdb-dataset

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env bash
+
+set -ueo pipefail
+
+usage() { echo 'Usage: PDBBOX=BOX_DIR pdb-dataset import --timeshift-to now DUMPFILE'; }
+
+if test "$#" -lt 1 -o -z "$PDBBOX"; then
+  usage 1>&2
+  exit 1
+fi
+
+misuse()
+{
+  usage 1>&2
+  exit 1
+}
+
+timeshift=''
+timeshiftsql=''
+dumpfile=''
+command=''
+while test $# -gt 0; do
+  case "$1" in
+    --timeshift-to)
+      test $# -gt 0 || misuse
+      timeshift="$2"
+      shift 2
+      ;;
+    *)
+      if [ -z $command ]; then
+        command="$1"
+        shift
+      elif [ -z $dumpfile ]; then
+        dumpfile="$1"
+        shift
+      else
+        echo "Unknown argument: $1" 1>&2
+        misuse
+      fi
+  esac
+done
+
+tmpdir="$(mktemp -d "pdb-dataset-XXXXXX")"
+tmpdir="$(cd "$tmpdir" && pwd -P)"
+trap "$(printf 'rm -rf %q' "$tmpdir")" EXIT
+
+case "$command" in
```

Comment:
Could we change this case to an if and break early? Also for readability we should enclose some ob these blocks into functions